### PR TITLE
Update French (fr) Essentials, Exploring, Use, Use cases, and Staking buckets

### PR DESCRIPTION
**Pull request by minimalsm**
## Description

Update French (fr) 

- Essentials
- Exploring
- Use Ethereum
- Use cases
- Staking

### CODE REVIEW

File: src/content/translations/fr/decentralized-identity/index.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
+---
+title: Identité décentralisée
+description: Qu’est-ce que l’identité décentralisée et pourquoi est-ce important ?
+lang: fr
+template: use-cases
+emoji: ":id:"
+sidebar: true
+sidebarDepth: 2
+image: ../../assets/eth-gif-cat.png
```
Comment (minimalsm) [inline src/content/translations/fr/decentralized-identity/index.md:9]: This isn't showing...

File: src/content/translations/fr/decentralized-identity/index.md [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+- [Qu'est-ce que l'identité décentralisée et pourquoi devriez-vous vous en préocupper ?](https://web3.hashnode.com/what-is-decentralized-identity) — _Emmanuel Awosika_
+
+### Vidéos {#videos}
+
+- [Identité décentralisée (Bonus Session Livestream)](https://www.youtube.com/watch?v=ySHNB1za_SE&t=539s) — _Une formidable vidéo explicative sur l'identité décentralisée par Andreas Antonopolous_
+- [Connexion avec Ethereum et Identité décentralisée avec Ceramic, IDX, React, et 3ID Connect](https://www.youtube.com/watch?v=t9gWZYJxk7c) — _tutoriel YouTube sur la création d'un système de gestion d'identité pour créer, lire et mettre à jour un profil d'utilisateur en utilisant son portefeuille Ethereum par Nader Dabit_
+- [BrightID - Identité décentralisée sur Ethereum](https://www.youtube.com/watch?v=D3DbMFYGRoM) — _Épisode de podcast non bancaire parlant de BrightID, une solution d'identité décentralisée pour Ethereum_
+- [The Off Chain Internet: Decentralized Identity & Verifiable Credentials](https://www.youtube.com/watch?v=EZ_Bb6j87mg) — EthDenver 2022 présentation de Evin McMullen
```
Comment (minimalsm) [inline src/content/translations/fr/decentralized-identity/index.md:179]:
```suggestion
- [The Off Chain Internet: Decentralized Identity & Verifiable Credentials](https://www.youtube.com/watch?v=EZ_Bb6j87mg) — _EthDenver 2022 présentation de Evin McMullen_
```

Comment (minimalsm) [reply src/content/translations/fr/decentralized-identity/index.md:179]: Title isn't translated.

Comment (minimalsm) [reply src/content/translations/fr/decentralized-identity/index.md:179]: Raised Crowdin issue

File: src/content/translations/fr/social-networks/index.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
+---
+title: Les réseaux sociaux décentralisés
+description: Un aperçu des réseaux sociaux décentralisés sur Ethereum
+lang: fr
+template: use-cases
+emoji: ":mega:"
+sidebar: true
+sidebarDepth: 2
+image: ../../assets/ethereum-learn.png
```
Comment (minimalsm) [inline src/content/translations/fr/social-networks/index.md:9]: Image not showing.

File: src/intl/fr/page-layer-2.json [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+  "layer-2-how-does-layer-2-work-2": "Une blockchain de couche 2 communique régulièrement avec Ethereum (en soumettant des paquets de transactions) afin de s'assurer qu'elle dispose de garanties de sécurité et de décentralisation similaires. Tout cela ne nécessite aucune modification du protocole de couche 1 (Ethereum). Cela permet à la couche 1 de gérer la securité, la disponibilité des données et la décentralisation, tandis que la couche 2 gère le passage à l'échelle du réseau. La couche 2 soulage la couche 1 de nombreuses transactions et inscrit les preuves finalisées sur la couche 1. En retirant cette charge de transations de la couche 1, la couche de base devient moins congestionnée et tout devient plus évolutif.",
+  "layer-2-rollups-title": "Rollups",
+  "layer-2-rollups-1": "Les rollups sont actuellement la solution de couche 2 préférée pour la mise à l'échelle d'Ethereum. En utilisant les rollups, les utilisateurs peuvent <a href=\"https://l2fees.info/\">réduire les frais de gas jusqu'à 100 fois</a>par rapport à la couche 1.",
```
Comment (minimalsm) [inline src/intl/fr/page-layer-2.json:47]:
```suggestion
  "layer-2-rollups-1": "Les rollups sont actuellement la solution de couche 2 préférée pour la mise à l'échelle d'Ethereum. En utilisant les rollups, les utilisateurs peuvent <a href=\"https://l2fees.info/\"> réduire les frais de gas jusqu'à 100 fois</a>par rapport à la couche 1.",
```

**Comment by minimalsm** [comment]
This is good other than the images not showing on the new use case pages.

File: src/content/translations/fr/social-networks/index.md (hunk 3 above)
Comment (minimalsm) [inline src/content/translations/fr/social-networks/index.md:9]:
```suggestion
image: ../../../../assets/ethereum-learn.png
```

File: src/content/translations/fr/decentralized-identity/index.md (hunk 1 above)
Comment (minimalsm) [inline src/content/translations/fr/decentralized-identity/index.md:9]:
```suggestion
image: ../../../../assets/eth-gif-cat.png
```

**Review by corwintines — APPROVED** [review]

**Pull request merged**